.github/draft-release-notes-config.yml

```diff
@@ -0,0 +1,36 @@
+# The overall template of the release notes
+template: |
+  Compatible with Elasticsearch (**set version here**) and Open Distro for Elasticsearch (**set version here**).
+  $CHANGES
+# Setting the formatting and sorting for the release notes body
+name-template: Version (set version here)
+change-template: '- $TITLE (PR #$NUMBER)'
+sort-by: merged_at
+sort-direction: ascending
```

Comment:
Made a few updates to the template to match the standards, will want to change to something like this: 
```
# The overall template of the release notes
template: |
  Compatible with Elasticsearch (**set version here**).
  $CHANGES

# Setting the formatting and sorting for the release notes body
name-template: Version (set version here)
change-template: '* $TITLE (#$NUMBER)'
sort-by: merged_at
sort-direction: ascending
replacers:
  - search: '##'
    replace: '###'
```

Reply:
Cool, thanks. Will update.